content/components/mermaid-diagrams.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Mermaid diagrams"
+description: "Display diagrams using Mermaid"
+icon: "chart-network"
+---
+
+<RequestExample>
+
+````md Mermaid Flowchart Example
+```mermaid
+  flowchart LR
+    subgraph subgraph1
+        direction TB
+        top1[top] --> bottom1[bottom]
+    end
+    subgraph subgraph2
+        direction TB
+        top2[top] --> bottom2[bottom]
+    end
+    %% ^ These subgraphs are identical, except for the links to them:
+
+    %% Link *to* subgraph1: subgraph1 direction is maintained
+    outside --> subgraph1
+    %% Link *within* subgraph2:
+    %% subgraph2 inherits the direction of the top-level graph (LR)
+    outside ---> top2
+```
+````
+
+</RequestExample>
+
+[Mermaid](https://mermaid.js.org/) lets you create visual diagrams using text and code. For example, the following code will render a Flowchart Mermaid diagram.
+
+````md
+```mermaid
+  flowchart LR
+    subgraph subgraph1
+        direction TB
+        top1[top] --> bottom1[bottom]
+    end
+    subgraph subgraph2
+        direction TB
+        top2[top] --> bottom2[bottom]
+    end
+    %% ^ These subgraphs are identical, except for the links to them:
+
+    %% Link *to* subgraph1: subgraph1 direction is maintained
+    outside --> subgraph1
+    %% Link *within* subgraph2:
+    %% subgraph2 inherits the direction of the top-level graph (LR)
+    outside ---> top2
+```
+````
```

Comment:
Don't need this. It's a duplicate since we already have it in the request example